.github/workflows/ci.yml

```
@@ -17,9 +17,10 @@ jobs:
     strategy:
       matrix:
         node-version: [10, 12, 14, 16]
+        redis-tag: [5,6]
```

Space after comma for consistency.
```suggestion
        redis-tag: [5, 6]
```